Mathlib/Algebra/Order/Group/Defs.lean

```diff
@@ -20,11 +20,6 @@ may differ between the multiplicative and the additive version of a lemma.
 The reason is that we did not want to change existing names in the library.
 -/
 
-/-
-`NeZero` should not be needed at this point in the ordered algebraic hierarchy.
--/
-assert_not_exists NeZero
```

Comment:
Perhaps it would be good to `assert_not_exists not_neZero` here to check that `NeZero.lean` is not imported?